public/js/components/submit-button.jsx

```diff
@@ -7,10 +7,10 @@ import { gettext } from 'utils';
 export default class SubmitButton extends Component {
 
   static propTypes = {
-    cssModifier: PropTypes.string.isRequired,
+    cssModifier: PropTypes.string,
```

Comment:
I guess it's not fallout from this patch but I just noticed the `defaultProps` have the wrong property name for `cssModifier`